### PR TITLE
Media Library, table view, new column last modified date and time

### LINKED ISSUE #6291: Media Library: column for last modified date time
Hello guys,

Could interest to show last modified date time of a file in the media library and sort by this column? If yes, could I do a PR to add this functionality?

Thank you.

### PR DESCRIPTION
**Pull request by PiemP**
resolve #6291 

added a new column to media library table to allow user to sort by last modified date and time

- updated vuejs code to print the new column
- print date value by toLocaleString() method
- added the code to sort the table by the lastModify column
- added lastModify property to object returned by ContentFileResult method in the AdminController

### CODE REVIEW

**Comment by jptissot** [comment]
Made some updates to the media code in https://github.com/OrchardCMS/OrchardCore/pull/5906 will have small conflicts

**Comment by PiemP** [comment]
updated branch after #5906 merge

File: src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js [hunk 1]
```diff
@@ -352,7 +358,7 @@ function initializeMediaApplication(displayMediaApplication, mediaApplicationUrl
               return;
             }
 
-            confirmDialog(_objectSpread(_objectSpread({}, $("#deleteMedia").data()), {}, {
+            confirmDialog(_objectSpread({}, $("#deleteMedia").data(), {
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js:361]: Before it was the opposite ;) as i think after some updates a full gulp built was not done again

But now if you update node and npm, do npm install and then a gulp build the generated line is

    confirmDialog(_objectSpread(_objectSpread({}, $("#deleteMedia").data()), {}, {

File: src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js [hunk 2]
```diff
@@ -695,7 +701,7 @@ Vue.component('folder', {
         return;
       }
 
-      confirmDialog(_objectSpread(_objectSpread({}, $("#moveMedia").data()), {}, {
+      confirmDialog(_objectSpread({}, $("#moveMedia").data(), {
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js:704]: Idem

File: src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js [hunk 3]
```diff
@@ -267,7 +273,7 @@ function initializeMediaApplication(displayMediaApplication, mediaApplicationUrl
               return;
             }
 
-            confirmDialog(_objectSpread(_objectSpread({}, $("#deleteFolder").data()), {}, {
+            confirmDialog(_objectSpread({}, $("#deleteFolder").data(), {
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js:276]: Before it was the opposite ;) as i think after some updates a full gulp built was not done again

But now if you update node and npm, do npm install and then a gulp build the generated line is

   confirmDialog(_objectSpread(_objectSpread({}, $("#deleteFolder").data()), {}, {

File: src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js [hunk 4]
```diff
@@ -309,7 +315,7 @@ function initializeMediaApplication(displayMediaApplication, mediaApplicationUrl
               return;
             }
 
-            confirmDialog(_objectSpread(_objectSpread({}, $("#deleteMedia").data()), {}, {
+            confirmDialog(_objectSpread({}, $("#deleteMedia").data(), {
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js:318]: Idem

File: src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js (hunk 1 above)
Comment (PiemP) [reply src/OrchardCore.Modules/OrchardCore.Media/wwwroot/Scripts/media.js]: I hope should be ok 😅 ... thanks @jtkech

**Comment by sebastienros** [comment]
GTG if someone can ensure that the media libraries are up to date with other PRs that got merged in the meantime

**Pull request merged**